README.md

```diff
@@ -1,70 +1,169 @@
+- npm
 
-### Analyzing the Bundle Size
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Clone this repository to your desired folder:
 
-### Making a Progressive Web App
+```sh
+  cd my-folder
+  git clone https://github.com/pushkar1995/caps-metrics-app-react.git
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Install
 
-### Advanced Configuration
+Install the prerequisites this project using:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+```sh
+  cd caps-metrics-app-react
+  npm install
+  npm run build
+```
 
-### Deployment
+### Usage
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+To run the project:
 
-### `npm run build` fails to minify
+```sh
+  npm start
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Run tests
+
+To run tests after setting up the linters, run the following command:
+
+```sh
+  npx eslint "**/*.{js,jsx}"
+```
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+## 👥 Author <a name="authors"></a>
+
+👤 **Pushkar Gautam**
+
+- GitHub: [@pushkar1995](https://github.com/pushkar1995)
+- Twitter: [@Pushkar9951](https://twitter.com/Pushkar9951)
+- LinkedIn: [pushkargautam](https://www.linkedin.com/in/pushkar-gautam-94a385271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<br>
+
+-**In the future CSS transitions & animation will be added to the page, to have more inviting / interactive feel for the user**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>Contributions, issues, and feature requests are welcome!
+
+<br>
+
+Feel free to check the [issues page](https://github.com/pushkar1995/caps-metrics-app-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+<br>
+
+If you like my page please show your support and give this page a star!
+<br>
+<br>
+[![Star]](https://github.com/pushkar1995/caps-metrics-app-react)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<br>
+To start with i would like to thank Nelson Sakwa on Behance who created the design of the app and was kind to share it in order for others to learn.
+
+
```

Comment:
- [ ]  Hello, can you provide the link to navigate to `Nelson Sakwa` website? I think that could be the best way to acknowledge, also it will show more professionalism.

Reply:
Hello @OLIPLICHE thank you for the review. Just updated the README file with the link as you have mentioned.

Reply:
Hello @OLIPLICHE thank you for the review. Just updated the README file with the link as you have mentioned.

Reply:
This change has been done well

Reply:
@peterrobert thankyou for the review.